packages/plugin-dts/src/index.ts

```diff
@@ -79,7 +79,28 @@ export const pluginDts = (options: PluginDtsOptions): RsbuildPlugin => ({
         return;
       }
 
-      await Promise.all(dtsPromises);
+      promisesResult = await Promise.all(dtsPromises);
+    });
+
+    api.onAfterBuild({
+      handler: ({ isFirstCompile }) => {
+        if (!isFirstCompile) {
+          return;
+        }
+
+        for (const result of promisesResult) {
+          if (result.startsWith('Error')) {
```

Comment:
I know using a `string` works here. But what about using a structured object, e.g. `{ status: 'error' | 'success' }`, which is way easier to extend in the future.

Reply:
good idea